### PR TITLE
Update replication configuration for PG12+

### LINKED ISSUE #479: Replication configuration changes PostgreSQL 12 'How To' is now incorrect
# Describe the bug

Errors returned when following along on how to create a base backup. Instructions don't work for TimescaleDB built on PostgreSQL 12. 

Reported at https://timescaledb.slack.com/archives/C63MYDZ35/p1632578674120100

## What do the docs say now?

Use recovery.conf but when following along, these errors are encountered:

```
2021-09-25 13:55:50.562 UTC [1] LOG:  starting PostgreSQL 12.8 on x86_64-pc-linux-musl, compiled by gcc (Alpine 10.3.1_git20210424) 10.3.1 20210424, 64-bit
2021-09-25 13:55:50.562 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2021-09-25 13:55:50.562 UTC [1] LOG:  listening on IPv6 address "::", port 5432
2021-09-25 13:55:50.567 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2021-09-25 13:55:50.657 UTC [21] LOG:  database system was interrupted; last known up at 2021-09-25 13:45:43 UTC
2021-09-25 13:55:55.262 UTC [21] FATAL:  using recovery command file "recovery.conf" is not supported
2021-09-25 13:55:55.263 UTC [1] LOG:  startup process (PID 21) exited with exit code @1000101
```

## What should the docs say?

The issue along with the correct approach for PostgreSQL 12 is described in an article https://www.2ndquadrant.com/en/blog/replication-configuration-changes-in-postgresql-12/ 

# Page affected

https://docs.timescale.com/timescaledb/latest/how-to-guides/replication-and-ha/replication/#create-a-base-backup-on-the-replica

# Version affected

Current

# Subject matter expert (SME)

# Screenshots

# Any further info

Thanks to @MelchiSalins for the report.

### PR DESCRIPTION
**Pull request by charislam**
# Description

Replication configuration instructions don't work for PG12+ because PG12 removed the `recovery.conf` file and moved its parameters to `posgresql.conf` plus a new `standby.signal` or `recovery.signal` file. This PR updates the replication instructions to work for recent PG versions.

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes #479

### CODE REVIEW

**Review by Loquacity — APPROVED** [review]

File: timescaledb/how-to-guides/replication-and-ha/configure-replication.md [hunk 1]
```diff
@@ -163,37 +163,35 @@ from a base backup of the primary instance.
     ```
     The -W flag prompts you for a password. If you are using this command in an
     automated setup, you might need to use a [pgpass file][pgpass-file].
-1.  When the backup is complete, create a [recovery.conf][postgres-recovery-docs]
+1.  When the backup is complete, create a [standby.signal][postgres-recovery-docs]
     file in your data directory, and set the appropriate permissions.
```
Comment (mkindahl) [inline timescaledb/how-to-guides/replication-and-ha/configure-replication.md:167]: What are the appropriate permissions?

Comment (mkindahl) [reply timescaledb/how-to-guides/replication-and-ha/configure-replication.md:167]: Ah, it is given below, but might be good to write here as well.

**Review by mkindahl — APPROVED** [review]

File: timescaledb/how-to-guides/replication-and-ha/configure-replication.md (hunk 1 above)
Comment (charislam) [reply timescaledb/how-to-guides/replication-and-ha/configure-replication.md:167]: Absolutely! Thanks @mkindahl. Just want to double check with you that `0600` is the appropriate permissions level? Those were the permissions for `recovery.conf`, but after trawling the PostgreSQL docs, I've seen nothing about whether that's still the case for `standby.signal`.

Comment (charislam) [inline timescaledb/how-to-guides/replication-and-ha/configure-replication.md:167]:
```suggestion
    file in your data directory, and set user read and write permissions. 
```

**Review by Loquacity — APPROVED** [review]

**Pull request merged**